plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -514,5 +518,82 @@ contract('PaymentExitGame - End to End Tests', ([_, richFather, bob]) => {
                 });
             });
         });
+
+        describe('should process in-flight exits', () => {
```

Comment:
I might follow the style with other tests, like
```
Given Alice deposited with ETH
   Given a in-flight tx not mined
        The in-flight tx should be able to in-flight exit
             And then the output owner piggybacks
                  And then someone processes the exits for ETH after a week
```

Reply:
done